src/Runner.Listener/CommandSettings.cs

```diff
@@ -169,6 +169,15 @@ public string GetRunnerName()
                 validator: Validators.NonEmptyValidator);
         }
 
+        public string GetRunnerGroupName(string defaultPoolName = null)
+        {
+            return GetArgOrPrompt(
+                name: Constants.Runner.CommandLine.Args.RunnerGroup,
+                description: "Enter the name of the runner group to add this runner to:",
```

Comment:
Would it be worth listing the available runner groups that the user can choose?

Reply:
So we didn't in the past. I think the problem is around how many runner groups there are in the tenant.